bin/proof-generator/src/arg_parser.cpp

```diff
@@ -71,12 +71,13 @@ namespace nil {
             // clang-format off
             auto options_appender = config.add_options()
                 ("proof,p", make_defaulted_option(prover_options.proof_file_path), "Output proof file")
+                ("json,json", make_defaulted_option(prover_options.json_file_path), "JSON proof file")
```

Comment:
Second _json_ in `json,json` makes no sense. Second value is a short arg name you could use intead of the long one. You could either remove second part or make it `j`, since it is unused for now.
Btw, do we need to generate both files during a single run? Mb create a flag which will set either we need regural proof or json one?